### DISCUSSION TITLE
Creating UTM Virtualized Ubuntu VM

### DISCUSSION THREAD

**Discussion by Pikishti** [utmapp/UTM#5044] [UTM Help (macOS)]
Hello everyone,

I am having issues with the booting of a VM that uses Ubuntu. I am using the UTM app in an M1 Macbook Pro. In the image you can see the settings for the VM machine and also the issue I am getting when trying to start it. 
<img width="1331" alt="UTM issue" src="https://user-images.githubusercontent.com/114188730/220202361-eaa000c9-b7cb-4377-81ab-8f6a2d5660f6.png">

Has anyone ever experienced this issue? How can I fix it?

**Comment by gitblanc** [comment]
There is a related question on official ubuntu web: https://askubuntu.com/questions/954042/why-does-computer-show-device-mapping-table-on-efi-shell-after-ubuntu-installati
Hope it serves you

**Comment by jdork0** [comment]
You have arch/machine type set to arm64, but you appear to be using the amd64 iso.  Try an arm64 iso.
